apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/controller/UserInfoController.java

```diff
@@ -76,6 +76,16 @@ public void createOrUpdateUser(@RequestBody UserPO user) {
     }
   }
 
+  @PreAuthorize(value = "@permissionValidator.isSuperAdmin()")
+  @PostMapping("/users/enabled")
```

Comment:
It would be better to use `PUT` here.
```suggestion
  @PutMapping("/users/enabled")
```